bindings/python/docs/Orbit Determination.ipynb

```diff
@@ -0,0 +1,307 @@
+{
```

Comment:
@apaletta3 I am assuming you're pushing those to show future implementation? same for the GMAT validation files?

Reply:
Yup this is correct, but the GMAT validation files are used for running the current plotting jupyter notebooks for propagation, not orbit determination.